addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -766,7 +775,7 @@
         </div>
         <div class="card-footer bg-white text-600">
             <div class="d-flex align-items-center small">
-                <span class="fw-bold me-auto" t-field="slide.completion_time" t-options='{"widget": "float_time"}'/>
+                <span class="fw-bold me-auto" t-field="category['category'].completion_time" t-options='{"widget": "float_time"}'/>
```

Comment:
Hmm is this correct? Not sure why we use the category completion now instead of the slide completion for this part.